Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -1243,8 +1252,56 @@ namespace Multiplayer
     }
     AZ_CONSOLEFREEFUNC(host, AZ::ConsoleFunctorFlags::DontReplicate, "Opens a multiplayer connection as a host for other clients to connect to");
 
-    void connect([[maybe_unused]] const AZ::ConsoleCommandContainer& arguments)
+    void launch_local_client(const AZ::ConsoleCommandContainer& arguments)
     {
+        // Try finding the game launcher from the executable folder where this server was launched from.
+        AZ::IO::FixedMaxPath gameLauncherPath = AZ::Utils::GetExecutableDirectory();
```

Comment:
Does this work with an installer?

Reply:
good question, was going to try this with the Linux installer; currently the INSTALL target is unknown and the online deb package failed install. I'll keep looking for a way to test this...
This should only be called on a gamelauncher or serverlauncher, not editor, so I think those usually typically live in the same folder even if using the Installer

Reply:
Works with INSTALL! but this comment reminded me i should turn off the HOST option when in Editor :P Updated!